Examples/YouTubeClone/Composer/YTChatComposerViewController.swift

```diff
@@ -9,13 +9,17 @@ import UIKit
 final class YTChatComposerViewController: ComposerVC {
     // We need to cast the composerView to our new `YTChatMessageComposerView`
     // so that we can have access to the new `emojiButton`.
-    var ytMessageComposerView: YTChatMessageComposerView {
-        composerView as! YTChatMessageComposerView
+    var ytMessageComposerView: YTChatMessageComposerView? {
+        composerView as? YTChatMessageComposerView
     }
     
     override func setUp() {
         super.setUp()
 
+        guard let ytMessageComposerView = ytMessageComposerView else {
+            return
+        }
```

Comment:
In that case, I would be using `if` instead of the early exist because if we put more code for other UI components but this guard fails, we will never reach the code for other components

Reply:
Yep good suggestion 👍🏼